web/main/urls.py

```diff
@@ -105,6 +106,9 @@
     # path('pages/privacy-policy/', RedirectView.as_view(pattern_name='privacy-policy', permanent=True)),
     # path('pages/terms-of-service/', RedirectView.as_view(pattern_name='terms-of-service', permanent=True)),
     # path('pages/faq/', RedirectView.as_view(pattern_name='faq', permanent=True)),
+
+    # redirects
+    path('subscribe', lambda request: redirect('https://opencasebook.us1.list-manage.com/subscribe?u=894c9c0d9d7d7c5f8de8d76aa&id=89f2a23e12')),
```

Comment:
The redirect will be at `https://opencasebook.org/subscribe` and redirect to `https://opencasebook.us1.list-manage.com/subscribe?u=894c9c0d9d7d7c5f8de8d76aa&id=89f2a23e12` .